### DISCUSSION TITLE
Duplicate grpc.health.v1.Health error on upgrading starter to 4.5.10

### DISCUSSION THREAD

**Discussion by debraj-manna** [LogNet/grpc-spring-boot-starter#303] [General]
I am getting the below error on upgrading starter to 4.5.10 with spring-boot 2.5.8.

```
 Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'exceptionHandlerMethodResolver' defined in class path resource [org/lognet/springboot/grpc/autoconfigure/GRpcAutoConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.lognet.springboot.grpc.recovery.GRpcExceptionHandlerMethodResolver]: Factory method 'exceptionHandlerMethodResolver' threw exception; nested exception is java.lang.IllegalStateException: Duplicate key grpc.health.v1.Health (attempted merging values com.spotnana.healthcheck.GrpcHealthCheckService@4f5c30b1 and org.lognet.springboot.grpc.health.DefaultHealthStatusService$$EnhancerBySpringCGLIB$$76cc08ef@4d0e1a9a)
 ```

Any one faced the same problem?

**Comment by jvmlet** [comment]
Looks like you are registering another health check service `com.spotnana.healthcheck.GrpcHealthCheckService`

**Comment by debraj-manna** [comment]
Yes can I add only `GrpcHealthCheckService` and disable `DefaultHealthStatusService` ?

**Reply by jvmlet** [reply]
I suggest you to disable `com.spotnana.healthcheck.GrpcHealthCheckService` (don't register it)

**Reply by debraj-manna** [reply]
Is there a way to disable the `DefaultHealthStatusService` ?

**Comment by jvmlet** [comment]
You can exclude https://github.com/LogNet/grpc-spring-boot-starter/blob/master/grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/autoconfigure/consul/ConsulGrpcAutoConfiguration.java autoconfiguration using springs standard property.
